### PR TITLE
P3 => Tell me about your opinion

### PR DESCRIPTION
**Pull request by bahram700**
What should I add as a text?

### CODE REVIEW

File: f2.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+iopihjkhdsfsbnv
```
Comment (bahram5570) [inline f2.txt:1]: Try your name

Comment (bahram700) [reply f2.txt:1]: Like Bahram700?

Comment (bahram5570) [reply f2.txt:1]: Yeah, why not?

Comment (bahram700) [reply f2.txt:1]: OK, thanks.

**Comment by bahram5570** [comment]
I saw that

**Pull request merged**